public/mocks/v1_get_place_node_attributes.json

```diff
@@ -0,0 +1,767 @@
+            ]
+          }
+        ]
+      },
+      {
+        "name": "Socio-economic indicators",
+        "includedColumns": [
+          {
+            "name": "Soy production (2015)",
+            "unit": "Tn"
+          },
+          {
+            "name": "Cattle",
+            "unit": "heads"
+          },
+          {
+            "name": "Soy yield",
+            "unit": "Tn/ha"
+          },
+          {
+            "name": "Area soy",
+            "unit": "%"
+          },
+          {
+            "name": "Area maize",
+            "unit": "%"
+          },
+          {
+            "name": "Area cotton",
+            "unit": "%"
+          }
+        ],
+        "rows": [
+          {
+            "name": "Score",
+            "values": [
+              null,
+              2.0,
+              29.55,
+              0.0,
+              0.0,
+              null,
+              1840.0,
+              0.69,
+              2.0
+            ]
+          },
+          {
+            "name": "State Ranking",
+            "values": [
+              13,
+              27,
+              4,
+              51,
+              70,
+              6,
+              30
+            ]
+          }
+        ]
+      }
+    ],
+    "trajectory_deforestation": {
+      "includedYears": [
+        2003,
+        2004,
+        2005,
+        2006,
+        2007,
+        2008,
+        2009,
+        2010,
+        2011,
+        2012,
+        2013,
+        2014,
+        2015
+      ],
+      "lines": [
+        {
+          "name": "Soy related deforestation",
+          "type": "area",
+          "style": "area-pink",
+          "values": [
+            150790.0,
+            220800.0,
+            335576.0,
+            461100.0,
+            449500.0,
+            534240.0,
+            574308.0,
+            709500.0,
+            841427.0,
+            882126.0,
+            840000.0,
+            970536.0,
+            1017600.0
+          ]
+        },
+        {
+          "name": "Potential Soy related deforestation",
+          "type": "line",
+          "style": "line-dashed-pink",
+          "values": [
+            140290.0,
+            220100.0,
+            332576.0,
+            451100.0,
+            419500.0,
+            524240.0,
+            544308.0,
+            689500.0,
+            821427.0,
+            852126.0,
+            820000.0,
+            910536.0,
+            1010000.0
+          ]
+        },
+        {
+          "name": "Territorial Deforestation",
+          "type": "area",
+          "style": "area-black",
+          "values": [
+            100290.0,
+            200100.0,
+            302576.0,
+            421100.0,
+            409500.0,
+            514240.0,
+            504308.0,
+            609500.0,
+            841427.0,
+            822126.0,
+            800000.0,
+            900536.0,
+            1000000.0
+          ]
+        },
+        {
+          "name": "State Average",
```

Comment:
This are the values for all the municipalities of the state, averaged. So if you're looking at Querencia, average of all municipalities of the Mato Grosso state.